_maps/map_files/MetaStation/MetaStation_skyrat.dmm

```diff
@@ -24083,7 +24039,7 @@
 /obj/effect/decal/cleanable/dirt/dust,
 /obj/structure/closet/cardboard,
 /obj/effect/spawner/lootdrop/maintenance/two,
-/obj/item/toy/plush/lizard_plushie/green{
+/obj/item/toy/plush/lizardplushie{
```

Comment:
Mm, I changed that a couple of days ago

Reply:
yeah that one i dont know why it  happened either